tests/integration_test.py

```diff
@@ -1444,7 +1444,7 @@ def test_resource_warnings(self):
 
 class TestRegressions(unittest.TestCase):
     def setUp(self):
-        self.client = docker.client.Client(timeout=5)
+        self.client = docker.Client(timeout=5, base_url=DEFAULT_BASE_URL)
```

Comment:
Per https://github.com/tianon/docker-py/blob/fix-host-assumptions/tests/integration_test.py#L50, all of the timeouts are set to 5 - any longer probably indicates a problem with the test anyway. Did you have any particular issue with this, or just curiosity?